R/RLearner_regr_gbm.R

```diff
@@ -4,25 +4,26 @@ makeRLearner.regr.gbm = function() {
     cl = "regr.gbm",
     package = "gbm",
     par.set = makeParamSet(
-      makeDiscreteLearnerParam(id = "distribution", default = "gaussian", values = c("gaussian", "laplace", "poisson", "tdist", "quantile")),
-      # FIXME default for distribution in gbm() is bernoulli
+      makeDiscreteLearnerParam(id = "distribution", default = "bernoulli", values = c("gaussian", "bernoulli", "huberized", "adaboost", "coxph", "pairwise", "laplace", "poisson", "tdist", "quantile")),
       makeIntegerLearnerParam(id = "n.trees", default = 100L, lower = 1L),
       makeIntegerLearnerParam(id = "cv.folds", default = 0L),
       makeIntegerLearnerParam(id = "interaction.depth", default = 1L, lower = 1L),
       makeIntegerLearnerParam(id = "n.minobsinnode", default = 10L, lower = 1L),
-      makeNumericLearnerParam(id = "shrinkage", default = 0.001, lower = 0),
+      makeNumericLearnerParam(id = "shrinkage", default = 0.1, lower = 0),
       makeNumericLearnerParam(id = "bag.fraction", default = 0.5, lower = 0, upper = 1),
       makeNumericLearnerParam(id = "train.fraction", default = 1, lower = 0, upper = 1),
       makeNumericLearnerParam(id = "alpha", default = 0.5, lower = 0, upper = 1,
         requires = quote(distribution == "quantile")),
       makeLogicalLearnerParam(id = "keep.data", default = TRUE, tunable = FALSE),
-      makeLogicalLearnerParam(id = "verbose", default = FALSE, tunable = FALSE)
+      makeLogicalLearnerParam(id = "verbose", default = FALSE, tunable = FALSE),
+      makeIntegerLearnerParam(id = "n.cores", default = 1, tunable = FALSE)
     ),
     par.vals = list(distribution = "gaussian", keep.data = FALSE),
     properties = c("missings", "numerics", "factors", "weights", "featimp"),
     name = "Gradient Boosting Machine",
     short.name = "gbm",
-    note = '`keep.data` is set to FALSE to reduce memory requirements, `distribution` has been set to `"gaussian"` by default.',
+    note = poaste0(collapse = "", c('`keep.data` is set to FALSE to reduce memory requirements, `distribution` has been set to `"gaussian"` by default.',
```

Comment:
Mentioned default value for `distribution` is incorrect. Same note on distribution as for classification learner here?